packages/cli/src/commonlib/codeFlowLogin.ts

```diff
@@ -149,9 +149,9 @@ export class CodeFlowLogin {
       await this.startServer(server, serverPort!);
       this.pca!.getAuthCodeUrl(authCodeUrlParameters).then(async (response: string) => {
         if (this.accountName == "azure") {
-          CliCodeLogInstance.necessaryLog(LogLevel.Info, `[${constants.cliSource}] ${azureLoginMessage}`);
+          CliCodeLogInstance.necessaryLog(LogLevel.Info, `[${constants.cliSource}] ${azureLoginMessage}${response}`);
```

Comment:
The code logic looks good to me, just better naming... `response` to `url`

Reply:
Fixed. Thanks.